cli/dcoscli/config/main.py

```diff
@@ -109,6 +109,38 @@ def _unset(name):
 
     return 0
 
+def _format_config(file_value, effective_value, name=None, envvar_name=None,
+                   left_pad_fmt=None):
```

Comment:
I decided to move all the logic into _format_config; including token obscuring. It seems to be ok.